### ISSUE TITLE
Error on Virtual Products?

### ISSUE THREAD

**Issue by tvanhuis** [five23/shipsync-community#19]
Hello,

I'm getting an error on checkout for Virtual products:

Fatal error: Call to a member function getError() on a non-object in /home/kline/public_html/app/code/community/IllApps/Shipsync/Model/Shipping/Package/Origins.php on line 135

I tried adding the "weight" attribute to my virtual products, but that did not solve the error.  Am I missing a setting in the configuration?  Works great for all other product types.  Thanks!

Tim

**Comment by tvanhuis** [comment]
error occurs for downloadable products also, just found out...

**Comment by five23** [comment]
Heya, what version of Magento are you running?

**Comment by tvanhuis** [comment]
1.8.0.0

**Comment by five23** [comment]
Yup, definitely a bug...   Magento introduced better product/item parsing logic in v1.5, deprecating my item handlers.  Will fix asap.

**Comment by devmca** [comment]
Hi,
Can you tell me the steps how to integrate Fedex in Magento 1.9?
I am new to Magento and not able to integrate Fedex in Magento. Please help me. Thank you in advance.
